src/components/SideBar/index.tsx

```diff
@@ -1,14 +1,54 @@
 import React from 'react';
 
 import './styles.scss';
+import { connect } from 'react-redux';
+import { AppState } from '../../redux/reducer';
+import { AppAction, SectionName } from '../../redux/actionTypes';
+import changeSection from '../../redux/actions';
 
-const SideBar: React.FunctionComponent = () => (
-  <div className="menu-container">
-    <div className="app-name">Helpe</div>
-    <div className="section-text">Bug Reports</div>
-    <div className="section-text selected-section">Customer Service</div>
-    <div className="section-text">Feature Requests</div>
-  </div>
-);
+class SideBar extends React.Component<any> {
```

Comment:
is the reason this isn't `React.FunctionComponent` because you need to use `dispatch`?